komposable-architecture/build.gradle

```diff
@@ -1,6 +1,36 @@
 import com.toggl.komposable.buildsrc.Libs
 
-apply plugin: 'kotlin'
+plugins {
+    id 'kotlin'
+}
+
+ext {
+    PUBLISH_GROUP_ID = 'com.toggl'
+    PUBLISH_VERSION = '0.1.0'
+    PUBLISH_ARTIFACT_ID = 'komposable-architecture'
+    PUBLISH_DESCRIPTION = 'Kotlin implementation of Point-Free\'s composable architecture'
+    PUBLISH_URL = 'https://github.com/toggl/komposable-architecture'
+    PUBLISH_LICENSE_NAME = 'Apache License'
+    PUBLISH_LICENSE_URL =
+            'https://github.com/toggl/komposable-architecture/blob/main/LICENSE'
+    PUBLISH_DEVELOPER_ID = 'toggl'
+    PUBLISH_DEVELOPER_NAME = 'Toggl Track'
+    PUBLISH_DEVELOPER_EMAIL = 'tomas.valenta@toggl.com'
```

Comment:
What email address should I use here @pdrj ?

Reply:
Hey, the developer entry accepts multiple developers o/ Let's add the whole team o/

Reply:
Regarding email, our goto is I think support@toggl.com 🤔 There used to be something for mobile team, but I don't know how to access it 😄 If we clear this with support (they would just need to let us know about incoming emails), I think the support one would work.

Reply:
> Hey, the developer entry accepts multiple developers o/ Let's add the whole team o/

I changed that to our team, for now, I'm not sure who to include but we can definitely do that later.